.github/actions/install-r/action.yml

```diff
@@ -0,0 +1,33 @@
+name: "Setup Rig, R, and R Packages"
+description: "Install a specified R version using rig, with an option to install additional R packages."
+inputs:
+  version:
+    description: "The R version to install (e.g., 4.4.0)"
+    required: false
+    default: "4.4.0"
+runs:
+  using: "composite"
+  steps:
+    - name: Install Rig and R
+      shell: bash
+      env:
+        R_VERSION: "${{ inputs.version }}"
+      run: |
+        echo "Installing R version $R_VERSION using Rig..."
+        curl -Ls https://github.com/r-lib/rig/releases/download/latest/rig-linux-"$(arch)"-latest.tar.gz | sudo tar xz -C /usr/local
+        rig add "$R_VERSION"
+
+    - name: Check R Installation
+      shell: bash
+      run: |
+        R --version
+
+    - name: Conditionally Install R Packages
```

Comment:
Out of curiosity, why is this called out as conditional?

Reply:
oh! i can remove that. i thought the pkgs weren't always needed in some cases, but i think they are. let me update that.